tests/Sdk/Unit/Trace/TracerProviderTest.php

```diff
@@ -21,4 +25,41 @@ public function gettingSameTracerMultipleTimesShouldReturnSameObject()
 
         self::assertSame($tracer1, $tracer2);
     }
+
+    /**
+     * @test
+     */
+    public function newTraceProviderDefaultsToAlwaysOnSampler()
+    {
+        $traceProvider = new TracerProvider();
+        $description = $traceProvider->getSampler()->getDescription();
+
+        self::assertSame($description, 'AlwaysOnSampler');
```

Comment:
Should we potentially test for something else besides the `$description` here?

Reply:
I had thought that since the samplers already had unit tests verifying the logic they do regarding samplingResult and decisions, the main point of this would be to check that now we are using it.  I can add more, but it would probably be really similar to what the sampler unit tests do.  Let me know if you want that added.

Reply:
Was just curious on the reasoning.  This is acceptable to me!